ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -67,28 +65,12 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
   });
 
   @action
-  resetOptInModal() {
-    this.showActivateSecretsSyncModal = false;
-    this.hasConfirmedDocs = false;
+  onModalError(errorMsg: string) {
+    this.activationError = errorMsg;
   }
 
-  @task
-  @waitFor
-  *onFeatureConfirm() {
-    // must return null instead of root for non managed cluster.
-    // child namespaces are not sent.
-    const namespace = this.args.isHvdManaged ? 'admin' : null;
-    try {
-      yield this.store
-        .adapterFor('application')
-        .ajax('/v1/sys/activation-flags/secrets-sync/activate', 'POST', { namespace });
-      // must refresh and not transition because transition does not refresh the model from within a namespace
-      yield this.router.refresh();
-    } catch (error) {
-      this.error = errorMessage(error);
-      this.flashMessages.danger(`Error enabling feature \n ${errorMessage(error)}`);
-    } finally {
-      this.resetOptInModal();
-    }
+  @action
+  resetOptInModal() {
```

Comment:
I don't see this method called anywhere since `onFeatureConfirm` was moved to the component - can we remove it?

Reply:
we sure can! thanks for catching that 🧹